### ISSUE TITLE
Chat prompt responses and updates are not screen reader friendly

### ISSUE THREAD

**Issue by Tanvez** [Harvard-University-iCommons/chainlit#19]
| IMPACT      | URL          | ISSUE DESCRIPTION   | WCAG CRITERIA | TESTING METHOD | SCREENSHOT or SNIPPET | ACTION |
| :----:          |    :----:      |      :----:                        |  :----:                    | :----:                        | :----:                                     |  :----:    | 
| High    | n/a        | prompt responses and updates to the chat history are not announced to screen readers | 1.3.1 Info and Relationships - A | Manual - Safari/VO | https://share.getcloudapp.com/jkuRkJE8 | Use a live region to announce new messages.
